src/EditorFeatures/Core/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -179,12 +179,8 @@ internal sealed class CompletionSource : IAsyncExpandingCompletionSource
             var roslynTrigger = Helpers.GetRoslynTrigger(trigger, triggerLocation);
 
             // The completion service decides that user may want a completion.
-            if (completionService.ShouldTriggerCompletion(document.Project, document.Project.LanguageServices, sourceText, triggerLocation.Position, roslynTrigger, options, document.Project.Solution.Options))
-            {
-                return true;
-            }
-
-            return false;
+            return completionService.ShouldTriggerCompletion(
+                document.Project, document.Project.LanguageServices, sourceText, triggerLocation.Position, roslynTrigger, options, document.Project.Solution.Options, _roles);
```

Comment:
Unrelated to the merge, but we didn't pass in TextView roles to the method call, which is fixed here